pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
 flake8-ignore = E501
-addopts = --flake8
+addopts = --flake8 --doctest-modules
```

Comment:
seems `--doctest-modules` causes the failure we're seeing on Travis:
```
nbstripout.py:106: in <module>
    input_stream = io.TextIOWrapper(sys.stdin.buffer, encoding='utf-8')
E   AttributeError: 'DontReadFromInput' object has no attribute 'readable'
```

Reply:
up to you about this - don't know how you would prefer to test this

Reply:
@kynan any update/thoughts?

Reply:
OK I've now removed `--doctest-modules` and added `tests/test_functions.py` which does `impoert nbstripout` which still results in this error.

I'm quite sure this is a bug in `nbstripout`: `io.TextIOWrapper(sys.stdin.buffer, encoding='utf-8')` is not safe.

Reply:
delayed to later (https://github.com/kynan/nbstripout/pull/92/commits/4edbb8e2e5c4fccdf7120d7e9c0ba64902a30fcb)